Packs/JamfProtect/Integrations/JamfProtectEventCollector/JamfProtectEventCollector.py

```diff
@@ -461,18 +597,23 @@ def fetch_events(client: Client, max_fetch_alerts: int, max_fetch_audits: int, s
     audit_events, audit_next_run = [], {}
     alert_next_page = last_run.get("alert", {}).get("next_page", "")
     audit_next_page = last_run.get("audit", {}).get("next_page", "")
+    computer_next_page = last_run.get("computer", {}).get("next_page", "")
 
-    if not (alert_next_page or audit_next_page) or alert_next_page:
-        # The only case we don't trigger the alert event type cycle is when have only the audit next page token.
+    if not (alert_next_page or audit_next_page or computer_next_page) or alert_next_page:
+        # The only case we don't trigger the alert event type cycle is when have only the audit and computer next page token.
         alert_events, alert_next_run = get_events_alert_type(client, start_date_arg, max_fetch_alerts, last_run)
-    if not (alert_next_page or audit_next_page) or audit_next_page:
-        # The only case we don't trigger the audit event type cycle is when have only the alert next page token.
+    if not (alert_next_page or audit_next_page or computer_next_page) or audit_next_page:
+        # The only case we don't trigger the audit event type cycle is when have only the alert and computer next page token.
         audit_events, audit_next_run = get_events_audit_type(client, start_date_arg, end_date_arg, max_fetch_audits, last_run)
-    next_run: Dict[str, Any] = {"alert": alert_next_run, "audit": audit_next_run}
-    if "next_page" in (alert_next_run | audit_next_run):
+    if not (alert_next_page or audit_next_page or computer_next_page) or computer_next_page:
+        # The only case we don't trigger the computer event type cycle is when have only the alert and audit next page token.
+        computer_events, computer_next_run = get_events_computer_type(
+            client, start_date_arg, end_date_arg, max_fetch_computer, last_run)
+    next_run: Dict[str, Any] = {"alert": alert_next_run, "audit": audit_next_run, 'computer': computer_next_run}
+    if "next_page" in (alert_next_run | audit_next_run | computer_next_run):
         # Will instantly re-trigger the fetch command.
         next_run["nextTrigger"] = "0"
-    return alert_events, audit_events, next_run
+    return alert_events, audit_events, computer_events, next_run
```

Comment:
Four outputs make it a bit complex, can we use a NamedTuple for the various events, to ease the access later? 

e.g. 
```python
class EventResult(NamedTuple):
    alert_events: list[dict]
    audit_events: list[dict]
    computer_events: list[dict]
```